ui/decredmaterial/select.go

```diff
@@ -10,7 +10,7 @@ type Select struct {
 	size     float32
 	changed  bool
 	open     bool
-	selected int
```

Comment:
Why is this changed?

Reply:
It was not changed.

Reply:
I don't see why it would be necessary to change it in code when pretty much only the user changes it